membership_initial_fee/models/account_invoice.py

```diff
@@ -47,10 +47,13 @@ def initial_fee_create_check(self, product=False):
                 product.initial_fee == 'none' or
                 self.invoice_id.type != "out_invoice"):
             return False
+        # If we are associated to another partner membership, evaluate that
+        # partner lines
+        partner = self.partner_id.associate_member or self.partner_id
```

Comment:
?
```suggestion
        partner = self.partner_id.associate_member or self.invoice_id.partner_id
```

Reply:
There's a related field in the invoice line: https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_invoice.py#L1760-L1761

Reply:
I don't have good experiences with this kind of fields being filled in the proper time, so if you don't mind...

Reply:
Ok them :+1: